pyomo/contrib/mindtpy/mip_solve.py

```diff
@@ -225,6 +297,19 @@ def handle_master_mip_max_timelimit(master_mip, solve_data, config):
 
 
 def handle_master_mip_unbounded(master_mip, solve_data, config):
+    """
+    This function handles the result of the latest iteration of solving the MIP problem given an unbounded solution
+    due to the relaxation.
+
+    Parameters
+    ----------
+    master_mip: Pyomo model
+        the MIP master problem
+    solve_data: MindtPy Data Container (TODO: This is not used in this function, should it be removed?)
```

Comment:
I'd say that since the other two functions of handling MIP problems already follow a pattern, let's leave it. On the other hand, where the function is called we can just not pass anything in that argument